Code/src/test/java/nl/utwente/group10/ui/components/blocks/FunctionBlockTest.java

```diff
@@ -20,9 +20,10 @@
      */
     @Before
     public void setUp() throws Exception {
+        //TODO: not entirely correct.
```

Comment:
Could you just call your function 'foo' instead of '(+)'? Then you wouldn't have to deal with typeclasses.